src/config/test_postake_holy_grail.mlh

```diff
@@ -12,7 +12,7 @@
 
 [%%define genesis_ledger "test_five_even_stakes"]
 [%%define genesis_state_timestamp "2019-01-30 12:00:00-08:00"]
-[%%define block_window_duration 1500]
+[%%define block_window_duration 2000]
```

Comment:
why are you changing this?

Reply:
The 1500 block_window_duration seems too short for holy_grail. Sometimes it crashes because of this, so I changed it.

Reply:
alright fair enough